runtime/doc/options.txt

```diff
@@ -4974,6 +4974,16 @@ A jump table for the options with a short description can be found at |Q_op|.
 	and there is a letter before it, the completed part is made uppercase.
 	With 'noinfercase' the match is used as-is.
 
+						*'isexpand'* *'ise'*
+'isexpand' 'ise'	string	(default: "")
+			local to buffer
+	Defines characters and patterns for completion in insert mode. Used by
+	the |complete_match()| function to determine the starting position for
+	completion. This is a comma-separated list of triggers. Each trigger
+	can be:
+	- A single character like "." or "/"
+	- A sequence of characters like "->", "/*", or "/**"
```

Comment:
Should there be a way to escape `,`, so that `,` can be used in trigger chars?

Reply:
makes sense. Should we also support deduplication?

Reply:
also  careful consideration, it should return a nested list, such as `ise=abc,c`  return `[ [0, 'abc'], [2, 'c']`

Reply:
added by using `\\,`

Reply:
Will it be better to use `copy_option_part()` to separate triggers?

Reply:
yep should be better

Reply:
Actually I was talking about the part in `f_complete_match()`

Reply:
updated